config/crd/bases/jenkins.io_jenkins.yaml

```diff
@@ -5272,6 +5276,10 @@ spec:
                     volumeSize:
                       type: string
                   type: object
+                proxyConfigurationEnabled:
```

Comment:
it needs better explanation: It seems that you are taking the http proxy configuration from global config; if it is the case, it should appear in the field documentation and in the documentation.

Reply:
done